frontend/app/globals.css

```diff
@@ -98,3 +98,522 @@ body {
   aspect-ratio: 1;
   clip-path: polygon(75% 6.7%, 100% 50%, 75% 93.3%, 25% 93.3%, 0% 50%, 25% 6.7%);
 }
+
```

Comment:
This file become really big
I think it could be a good idea to split it,
for example you could create a css file dedicated for all the "button" stuff